src/System.Runtime.Extensions/ref/System.Runtime.Extensions.Manual.cs

```diff
@@ -11,12 +11,12 @@ namespace System
 {
     public static partial class Math
     {
-        public const double PI = 3.14159265358979323846;
```

Comment:
This should not need to be a separate `System.Runtime.Extensions.Manual.cs` file anymore if you change this to the autogenerate round-trippable value. The `System.Runtime.Extensions.Manual.cs` file should be deleted and the constants defined in the auto-generated `System.Runtime.Extensions.cs` (ie it should look like the linked netstandard ref file).

It is ok to use the roundtripable value in the auto-generated ref file. The constants in the auto-generated ref files are  not meant to in the form that humans would write them.